### PR TITLE
Setup Docker compose

### PR DESCRIPTION
**Pull request by Akarys42**
This PR adds a docker compose setup to spin up postgres, the site and the bot. The site will be build locally since we don't use docker hub (actually I think we should have both setup even once we'll have docker hub in place).

#### Documentation

I tried to document the setup in the README, if anyone wants to rewrite it feel free, I actually don't really like the way it is right now. 
Also, in the setup instructions I didn't mentioned the `config.toml` file, because I think we should keep it as a file that can be auto-generated by the bot, since we'll have to have some form of autoconfiguration in order for us to keep our sanity.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  bot:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    depends_on:
+      - web
+    tty: true
+    environment:
+      BOT_APIKEY: "${API_KEY:-GStandsForGreatDude}"
+      BOT_TOKEN:
```
Comment (gdude2002) [inline docker-compose.yaml:37]: No newline at end of file

**Comment by gdude2002** [comment]
This more or less looks fine. Did you want me to redo the README?

**Comment by Akarys42** [comment]
Well, I don't really like how I described it right now, but I don't really know how to make it better, if you can do that that would be nice

**Comment by gdude2002** [comment]
We can do better post-launch once things are on dockerhub, but this is just fine for now.

**Pull request merged**